docs/DeveloperGuide.md

```diff
@@ -201,6 +201,31 @@ The `RemindCommand` is then executed by `LogicManager`. After testing all events
     * Cons: Difficult to test, as the current time is based on real time.
 
 
+### Linkcontact feature
+
+The link contact feature allows users to link a contact to an event.
+
+#### Implementation
+The `linkcontact` feature is facilitated by `Event` class.
+The feature is implemented as follows:
+* When a user adds an event, he can optionally add a contact to the event.
+* When a user edits an event, he can optionally add a contact to the event.
+* When a user edits an event, he can optionally change the contact of the event.
+* If the user specify a contact that does not exist in the contact list, the event will not be added/edited.
+* If the user specify a contact that already exists in the contact list, the event will be added/edited with the contact.
```

Comment:
Should this two lines use `specifies` instead of `specify`?

Reply:
Good catch Gloria! Fixed in a6bfc74196c2a63e69256fab78f741daaae014f7!